TESTS/netsocket/tls/tlssocket_endpoint_close.cpp

```diff
@@ -48,6 +48,10 @@ static nsapi_error_t _tlssocket_connect_to_daytime_srv(TLSSocket &sock)
         return err;
     }
 
+    TEST_ASSERT_EQUAL(NSAPI_ERROR_OK, sock.set_root_ca_cert(tls_global::cert));
+
+    sock.set_timeout(10000); // Set timeout for case TLSSocket does not support peer closed indication
```

Comment:
Don't we also need this in other functions connecting to various servers, for example [here](https://github.com/ARMmbed/mbed-os/blob/7e0fb8fab9eddce102a50a3ef511ecb68d607956/TESTS/netsocket/tls/main.cpp#L106)?

Reply:
I think there is some room for little refactoring in here to use common method for opening the server connection in different test cases, but I think that should be done as a separate task.